src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+$c-grey: #eee;
+$squareSize: 100px;
+$padding: 10px;
+$gap: 1px;
+$week: 7;
+$weekStartMON: 0;
+$dayNamesLeft: (
+  "mon": 1,
+  "tue": 2,
+  "wed": 3,
+  "thu": 4,
+  "fri": 5,
+  "sat": 6,
+  "sun": 7
+);
```

Comment:
I think you need to put these variables into the style/utils/_variables.scss

Reply:
For greater file structuring I've done like you suggested to me. Thank you. Now done!)

Reply:
Move variables to corresponding file.